SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -53,6 +53,10 @@ TMap<FString, FString> ClassPathToSchemaName;
 TMap<FString, FString> SchemaNameToClassPath;
 TMap<FString, TSet<FString>> PotentialSchemaNameCollisions;
 
+const FString SchemaDatabasePackagePath = TEXT("/Game/Spatial/SchemaDatabase");
```

Comment:
@jherd-improbable This is something similar to what you're touching atm right?

Reply:
It is, yeah, I've put the const paths in SpatialConstants.h though. Should be a simple conflict to resolve.